go-controller/pkg/libovsdbops/model_client.go

```diff
@@ -437,6 +437,11 @@ func (m *modelClient) Lookup(opModels ...operationModel) error {
 	return err
 }
 
+func (m *modelClient) LookupOps(ops []ovsdb.Operation, opModels ...operationModel) ([]ovsdb.Operation, error) {
```

Comment:
I think lookup will never return any ops, because lookup doesn't change db state. And that is why we have `(m *modelClient) Lookup(opModels ...operationModel) error`